server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -19,11 +19,14 @@
  */
 package org.sonar.server.issue.ws;
 
+import org.sonar.core.user.GroupDto;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.core.user.GroupDto'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)